packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -84,7 +106,7 @@ export interface HealthCheck {
    * The approximate number of seconds between health checks for an individual target.
    * Must be 5 to 300 seconds
    *
-   * @default 10 seconds if protocol is `GENEVE`, 35 seconds if target type is `lambda`, else 30 seconds
+   * @default - 10 seconds if protocol is `GENEVE`, 35 seconds if target type is `lambda`, else 30 seconds
```

Comment:
I have corrected the usage of hyphens in some @default descriptions. If it is preferable, I can separate this into a different PR.